cmd/describe/cluster/cmd.go

```diff
@@ -157,9 +157,11 @@ func run(cmd *cobra.Command, argv []string) {
 		phase = fmt.Sprintf("(%s)", cluster.Status().Description())
 	}
 
+	domainPrefix := cluster.DomainPrefix()
```

Comment:
@machi1990 I assume we're running a migration in CS to populate this for all existing clusters?

Reply:
No need for a migration because all existing clusters have it populated already under the `cd_name` column.
The domain_prefix API field maps to `cd_name` column.

Reply:
Thanks for the explanation!